scripts/dts/edtlib.py

```diff
@@ -550,6 +550,11 @@ def _init_prop(self, name, options):
             # 'category: optional' property that wasn't there
             return
 
+        enum = options.get("enum")
+        if enum and val not in enum:
+            _err("value ({}) for property ({}) is not in enumerated "
+                 "list {} for node {}".format(val, name, enum, self.name))
```

Comment:
`self.path` is better than `self.name`, as it gives the full path to the node.

Reply:
Changed.